data-node/pom.xml

```diff
@@ -698,7 +715,26 @@
                         </configuration>
                     </execution>
                     <execution>
-                        <id>install-required-opensearch-plugins-aarch64</id>
+                        <id>install-opensearch-plugin-gcs-x64</id>
```

Comment:
@bernd Could we somehow reuse the executions? There are now 4 very similar - with differences only in architecture and plugin name. Thanks!

Reply:
I think the only sane way is to write a custom Java class to do the work. That would also get rid of the non-portable bash invocations.

Similar to: https://github.com/Graylog2/graylog2-server/blob/b0d2224278d4e0fc96dc5607407727070d732cea/graylog2-server/pom.xml#L1102-L1121

Reply:
Thanks! I'll prepare something, in a different PR.

Reply:
Done in https://github.com/Graylog2/graylog2-server/pull/22309